src/components/navBar/components/DropdownMenu.js

```diff
@@ -7,7 +7,7 @@ import DropdownItemsMenu from './DropdownItemsMenu';
 
 const DropdownMenu = ({
   classes, handleButtonClickEvent, linkText, clickedEl, dropDownElements,
-  navBarstyling, externalLinksFlag, externalLinks,
+  navBarstyling, externalLinksFlag, externalLinks, index,
```

Comment:
Can we pass a default value for index as 'aboutMenu' so it wont break any of the previous implementations